gateleen-hook/src/main/resources/gateleen_hooking_schema_hook

```diff
@@ -131,6 +135,32 @@
                 "header",
                 "value"
             ]
-        }
+        },
+		"ProxyOptions": {
+			"properties": {
+				"type": {
+                    "type": "string",
+                    "enum": [
+                        "HTTP",
+                        "SOCKS4",
+                        "SOCKS5"
+                    ]
+				},
+				"host": {
+					"type": "string"
+				},
+				"port": {
+					"type": "integer",
+					"minimum": 0
+				},
+				"username": {
+					"type": "string"
+				},
+				"password": {
+					"type": "string"
+				}
+			},
+			"additionalProperties": false
+		}
```

Comment:
**Bad indentation:** Some of those lines are indented by tabs whereas rest of file uses 4 spaces.